AccountManager/src/main/java/com/instinctools/trainerships/services/impl/JPAAccountService.java

```diff
@@ -0,0 +1,49 @@
+package com.instinctools.trainerships.services.impl;
+
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import org.springframework.stereotype.Service;
+
+import com.instinctools.trainerships.HibernateUtil;
+import com.instinctools.trainerships.model.Account;
+import com.instinctools.trainerships.model.Client;
+import com.instinctools.trainerships.services.AccountService;
+
+@Service
+public class JPAAccountService implements AccountService {
```

Comment:
Based on what you doing in this "Service"  I would say that it not a service at all - it's a data access layer which works with only JPA/SQL. One of the next tasks will be to add support of NoSQL storage - and it should be done like implement some new Data access layer and just inject it to existing real services - In your case its impossible to complete because your service is hard bounded to SQL storage
So to summarize - you need one more layer - Data access layer with it own interfaces and etc.